<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>406</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Rejstracja</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(249, 198, 171, 255), stop:1 rgba(232, 109, 52, 255));
font: bold 20pt &quot;Routed Gothic&quot;;
border-radius: 35px;
}
#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(249, 198, 171, 255), stop:1 rgba(232, 109, 52, 255));
}
#button_register{
background-color: rgb(248, 222, 214);
border-radius: 20px;
font: bold 17pt &quot;Routed Gothic&quot;;
color: rgba(224,102,46,99)
}
#button_register:hover {
background-color: rgba(228, 78, 9, 255);
color: white
}
#label_Rejestracja{
border-radius: 35px;
font: bold 20pt &quot;Routed Gothic&quot;;
color:white
}
#label_Login{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_Haslo{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_P_Haslo{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_Imie{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_Nazwisko{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_Email{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_P_Email{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Rejestracja">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REJESTRACJA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Imie">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Imię:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nazwisko">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Nazwisko:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Login">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Haslo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Hasło:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_P_Haslo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string> Powtórz hasło:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Email">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_P_Email">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string> Powtórz email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fname">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pass">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pass2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_login">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sname">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_register">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STWÓRZ KONTO</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_login</tabstop>
  <tabstop>lineEdit_pass</tabstop>
  <tabstop>lineEdit_pass2</tabstop>
  <tabstop>lineEdit_fname</tabstop>
  <tabstop>lineEdit_sname</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_email2</tabstop>
  <tabstop>button_register</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
